src/plugins/intel_gpu/src/graph/include/pass_manager.h

```diff
@@ -185,6 +185,12 @@ class prepare_conv_eltw_read_write_opt : public base_pass {
     void conv_eltwise_read_write_opt(program& p, program_node* node);
 };
 
+class prepare_primitive_fusing_through : public base_pass {
```

Comment:
Do we really need a separate pass for it? Can't it be a part of common fusion pass?

Reply:
Sure, it can be a part of prepare_primitive_fusings, but wouldn't it be better to keep it as a separate pass to be able to review changes before/after this optimization?